recipe/build.sh

```diff
@@ -3,53 +3,98 @@
 # unset the SUBDIR variable since it changes the behavior of make here
 unset SUBDIR
 
-USE_NONFREE=no
-
 declare -a _CONFIG_OPTS=()
 
+# We choose the following flags because we don't enable NON-FREE features
 # We do not care what the defaults are as they could change. Be explicit
 # about every flag.
-if [[ ${USE_NONFREE} == yes ]]; then
-  _CONFIG_OPTS+=("--enable-nonfree")
-  _CONFIG_OPTS+=("--disable-gpl")
-  _CONFIG_OPTS+=("--disable-gnutls")
-  _CONFIG_OPTS+=("--enable-openssl")
-  # The Cisco GPL-compliant wrapper (you need to get your own binaries for this)
-  _CONFIG_OPTS+=("--enable-libopenh264")
-  # GPL-3.0
-  _CONFIG_OPTS+=("--enable-libx264")
-else
-  _CONFIG_OPTS+=("--disable-nonfree")
-  _CONFIG_OPTS+=("--enable-gpl")
-  _CONFIG_OPTS+=("--enable-gnutls")
-  # OpenSSL 3 will be Apache-licensed so we can revisit this later:
-  # https://github.com/openssl/openssl/commit/151333164ece49fdba3fe5c4bbdc3333cd9ae66d
-  _CONFIG_OPTS+=("--disable-openssl")
-  # The Cisco GPL-compliant wrapper (you need to get your own binaries for this)
-  _CONFIG_OPTS+=("--enable-libopenh264")
-  # GPL-3.0
+_CONFIG_OPTS+=("--disable-nonfree")
+_CONFIG_OPTS+=("--enable-gpl")
```

Comment:
```suggestion
_CONFIG_OPTS+=("--enable-gpl")
```
Are these features required? If build without it, ffmpeg can be liensed lgpl instead of gpl.
https://ffmpeg.org/doxygen/6.1/md_LICENSE.html

Reply:
this stems from how the last build was configured, namely the local var `USE_NONFREE` was set to `no`, which means `--enable-gpl` was set. I was then set on keeping this.
What would be the advantage of LGPL vs GPL?

Reply:
In very simple terms, LGPL is less strict (the L stands for **L**esser). Companies in general don't like GPL (read often outright prohibit any use of GPL dependencies) but sometimes authorized LGPL dependencies.

Reply:
there's no `--disable-nonfree` flag, just these:
```
Licensing options:
  --enable-gpl             allow use of GPL code, the resulting libs
                           and binaries will be under GPL [no]
  --enable-version3        upgrade (L)GPL to version 3 [no]
  --enable-nonfree         allow use of nonfree code, the resulting libs
                           and binaries will be unredistributable [no]
 ```
 +1 for LGPL

Reply:
[this](https://github.com/AnacondaRecipes/ffmpeg-feedstock/blob/57ca332013295a161307fa10afdae2197a56cc93/recipe/build.sh#L6) is the local USE_NONFREE var I was referring to and you can see how by default we were already enabling GPL code.

Reply:
I'm ok with making the license less restrictive by removing GPL code, by the way. I was trying to stick to what the recipe already did, even though I don't see an apparent rational behind certain decisions.